src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,15 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        text = text.replaceAll("[^\\p{L}\\p{Nd}]+", "").toLowerCase();
+        StringBuilder sb = new StringBuilder(text);
+        String reversedText = sb.reverse().toString().toLowerCase();
+        int count = 0;
+        for (int i = 0; i < text.length(); i++) {
+            if (reversedText.charAt(i) == text.charAt(i)) {
+                count++;
```

Comment:
@SviatoslavExpert you can simplify this)

Reply:
you may require some algorithm knowlegde

Reply:
but do not spend on this task too much time. Try to spent not more then 1 hour

Reply:
Fixed. Hope this improvement will work out.